docs/fundamentals/interface.rst

```diff
@@ -0,0 +1,108 @@
+
+Pipelines are executed by clicking the yellow play button in the bottom right of the workspace. In the window that appears, you can name the execution, select a computation platform, and select a storage platform. Computation platforms specify what the compute resources used for execution of the operations, such as `SciServer Compute <https://apps.sciserver.org/compute/>`_, will be. Supported storage platforms, such as endpoints with an S3-compatible API, are used to store intermediate and output data. The provided storage option will be used for storing both the output objects defined in the pipeline, as well as all files used in execution of the pipeline.
+
+.. figure:: execute_pipeline.png
+    :align: center
+    :width: 75%
+
+Executions
+----------
+.. figure:: executions_tab.png
+    :align: center
+    :width: 75%
+
+This view allows the review of previous pipeline executions. Clicking on any execution will display any plotted data generated by the pipeline, and selecting multiple executions will display all of the selected plots together. Clicking the provided links will open either the associated pipeline or a trace of the execution (shown below). The blue icon in the top right of every operation allows viewing the text output of that operation. The execution trace can be viewed during execution to check the status of a running job. During execution, the color of a operation indicates its current status. The possible statuses are:
+
+- **Dark gray**: Pending Execution
+- **Light gray**: Execution Queued
+- **Yellow**: Execution in Progress
+- **Orange**: Execution Cancelled
+- **Green**: Successfully Finished Execution
+- **Red**: Execution Failed
+
+.. figure:: execution_finished.png
+    :align: center
+    :width: 50%
+
+Resources
+---------
+.. figure:: resources_tab.png
+    :align: center
+    :width: 75%
+
+This view shows the resources available for use in pipelines. Different types of resources are made available through DeepForge extensions and enable the introduction of new concepts into the project. One such example is `deepforge-keras <https://github.com/deepforge-dev/deepforge-keras>`_ which enables users to make neural networks architectures with a custom visual editor. The created architectures can then be referenced and used by operations for tasks such as training.From this view, resources can be created, deleted, and renamed. Resources are arranged by the deepforge system and cannot by manually reordered.
```

Comment:
"training.From" - missing space.

Reply:
The lack of manual reordering support might be better suited for an FAQ rather than here as it is easier to get out of date though it could be fine here, too. If you want to keep it here, I would change "deepforge" -> "DeepForge".

Reply:
I agree that a FAQ would be a more appropriate location. Since there isn't one yet, but this is far from urgent enough information to necessitate creating one, I just removed it for now.